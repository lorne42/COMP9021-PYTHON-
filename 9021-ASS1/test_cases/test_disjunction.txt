"Sir David, Sir Francis or Sir Grace is a Knight," whispered
Sir Harry. However, Sir David said: "I, Sir Francis or Sir Grace is a Knave," which
made me confused. Maybe both of them could be right?