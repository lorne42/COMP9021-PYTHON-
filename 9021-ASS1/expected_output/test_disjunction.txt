The Sirs are: David Francis Grace Harry
There are 3 solutions.